pkg/lib/inventory/model/client.go

```diff
@@ -29,6 +29,8 @@ type DB interface {
 	Count(Model, Predicate) (int64, error)
 	// Begin a transaction.
 	Begin(...string) (*Tx, error)
+	// With transaction.
+	With(fn func(*Tx) error, labels ...string) error
```

Comment:
did you upgrade the library in the process or where did this change came from?

Reply:
in that case, it can be handy to document the git hash we "forked" from

Reply:
ahh yeah great point I took it from the main branch which is v0.11.0. Added README